authors/apps/comments/serializers.py

```diff
@@ -16,3 +16,25 @@ class LikeCommentSerializer(serializers.ModelSerializer):
     class Meta:
         fields = ('id', 'comment', 'user')
         model = LikeComment
+
+
+class CommentHistorySerializer(serializers.ModelSerializer):
+    """
+    Class Handling History of Comment Edited
+    """
+    parent = serializers.ReadOnlyField(source='commented_by.parent')
+
+    class Meta:
+        model = Comment
+        fields = ('id', 'body', 'created_at',
+                  'updated_at', 'parent', 'author', 'article')
+
+
+class CommentHistorySerializer(serializers.ModelSerializer):
+    """
+    Class Handling History of Comment Edited
+    """
+
+    class Meta:
+        model = EditHistory
+        fields = ('id', 'body', 'created_at', 'user')
```

Comment:
@moseskamira It appears that there are two classes with the same name. i.e.
```class CommentHistorySerializer(serializers.ModelSerializer):```

Reply:
Worked upon it